bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -59,6 +61,10 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
           || request.getRequestURI().endsWith("register.jsp")
           || request.getRequestURI().endsWith("login.jsp")) {
         if (getCurrentUserPermission() == Permission.USER) {
+            if (request.getRequestURI().endsWith("requests")) {
+                RequestDispatcher requestDispatcher = request.getRequestDispatcher("requests?illegalAccess=false");
+                requestDispatcher.forward(request, response);
+            }
```

Comment:
Formatting: fix the indentation here =)

Reply:
Formatted

Reply:
Also, do you mean illegalAccess=true here?  I don't see any case where you actually set it to true in this code?  I am guessing that Permission.USER is not high enough permission to view the requests, and that only Permission.ADMIN can do that... but I'm not sure.  If not, where is the case where you set it to false?